packages/block-library/src/audio/index.js

```diff
@@ -30,6 +30,7 @@ export const settings = {
 			attribute: 'src',
 		},
 		caption: {
+			type: 'string',
```

Comment:
We have some docs with samples very similar to this.
https://github.com/WordPress/gutenberg/blob/master/docs/block-api/attributes.md#text

I think it may be worth it to add `type: 'string',` in these docs.

Reply:
Added in 9a6276b